packages/snap-controller/src/Search/SearchController.ts

```diff
@@ -447,6 +454,29 @@ export class SearchController extends AbstractController {
 		return params;
 	}
 
+	createResultSchemaMapping = (request: SearchRequestModel, response: [MetaResponseModel, SearchResponseModel]) => {
```

Comment:
I think I'd prefer this not be a class method if possible - as I can see it going away at some point.